nvflare/fuel/flare_api/flare_api.py

```diff
@@ -431,11 +431,7 @@ def monitor_job(
             if not job_status:
                 raise InternalError(f"missing status in job {job_id}")
 
-            if job_status in [
-                RunStatus.FINISHED_EXECUTION_EXCEPTION.value,
-                RunStatus.FINISHED_COMPLETED.value,
-                RunStatus.FINISHED_ABORTED,
-            ]:
+            if job_status.startswith("FINISHED"):
```

Comment:
If not using `RunStatus` anymore needs to remove the import otherwise the style tests failed.

Reply:
Ah, good catch. Will change.

Reply:
Fixed